### DISCUSSION TITLE
Missing PK output values for some subjects

### DISCUSSION THREAD

**Discussion by mags0728** [Open-Systems-Pharmacology/Forum#1449] [Questions & Problems]
Hello! I’m performing dosing simulations for virtual infant population of 1000 subjects. When I download the PK parameter results for the entire population, there are NaN values for certain subjects. For example, I created a virtual population of 1 month to 6 month old using the PK Sim population database and when I download the PK parameter results .csv file, it has no Ctrough output for 238 subjects (see screenshot). 

Why is it not producing PK values for some subjects? Any help to resolve this would be greatly appreciated!

Thank you in advance. 

![NaN Github Post](https://user-images.githubusercontent.com/97306838/217579529-145de0bf-329e-4de1-8d1a-3e91988cbcd9.PNG)

**Comment by tobiasK2001** [comment]
Hi, are other PK parameters like Cmax, AUCinf affected as well?

**Comment by PavelBal** [comment]
Simulations for some individuals might fail (the sampled physiology values unfavorable conditions for the solver). In this case, no results are generated and therefore no PK parameters can be calculated.

**Comment by mags0728** [comment]
Hello, @tobiasK2001 yes, the other PK parameters are affected as well. 

@PavelBal thank you for the explanation, that's what I'm suspecting as well. It sounds like there might not be a way to circumvent this issue since the physiology values for my certain population are unfavorable. 

Is there an error log in PK-Sim or a way to see which physiological values causing these individuals to fail?

Thank you!

**Reply by msevestre** [reply]
@mags0728 
You could extract the individual to your PKSim project (using the id, for instance, 2) and run the individual simulation with this individual
You should be able to see what's going on in more details (e..g how are the curve looking, what are the parameters etc)
I suspect that some parameters are out of physiological bounds somehow

**Reply by mags0728** [reply]
@msevestre great idea, thank you!

**Reply by msevestre** [reply]
There is a function for this BTW (right click on the population).

**Reply by tobiasK2001** [reply]
Hi, in my case I used in my Simulation particle dissolution to describe absorption and found that some individuals failed to converge due to variability in GI parameters and limited absorption during the given GI transit time. In that case  it helped to joggle the solver setting, e.g. decrease absolute and relative tolerance and/or set use Jacobean to no. This will increase runtime of the simulation but decrease likelihood that individuals fails

![image](https://user-images.githubusercontent.com/25585248/218036011-c2e5d800-67f7-415f-a207-23f42b0abb50.png)
Not sure if your case si similar, but just in case.

Best, Tobias

**Reply by mags0728** [reply]
Thank you for sharing, @tobiasK2001! I will also try this - from my troubleshoot yesterday where I extracted individuals and did individual simulations, it seems that the issue is related to the aging function (I'm working in children < 2 years old, see screenshot). So when I took out aging for individual simulations, it worked. However, when I did population simulations again without aging, it still resulted in the same individuals producing no output for PK parameters...

So I will also try your suggestion to see if that will lessen the number of individuals that fail. 

![IND 6 - Fat-fraction error](https://user-images.githubusercontent.com/97306838/218101085-b293d1ef-63ae-4455-9535-9b153a476c62.PNG)

**Reply by msevestre** [reply]
ah but here you see some error. Percentile has become 0 which is not possible normally...

**Reply by mags0728** [reply]
@msevestre is there a way to make this percentile a non-zero value or is it intrinsic value that cannot be changed?

Thank you!

**Reply by Yuri05** [reply]
@mags0728 Can you upload your project here? If not: 
* how does the distribution of **Fat/Fraction Interstitial** in your population look like?
* what is the value of **Fat/Fraction Interstitial** in your "problematic" Individual?
<details><summary><b>Example: distribution of Fat fraction interstitial in a population</b></summary>

![grafik](https://user-images.githubusercontent.com/25061876/219127887-ec582832-259d-4468-998c-9d6190f19b3f.png)

</details>

**Reply by mags0728** [reply]
Hi @Yuri05, 

- Here's the Fat/Fraction Interstitial distribution for my population: 
![Fat fraction in population](https://user-images.githubusercontent.com/97306838/219168809-8a457e47-b7bb-42dc-8a03-4b8d515d64a1.PNG)

- The fat/fraction interstitial value is 0.53 for a problematic individual:
![Fat fraction in IND 6](https://user-images.githubusercontent.com/97306838/219169568-fe6320db-2a6d-4639-ba32-07e6a72defd1.PNG)

Thanks!

**Reply by Yuri05** [reply]
does not look wrong for me.
Is it possible to attach your project? 
If not: can you attach just the problematic individual from above (e.g. save it as a template; create new empty project and load the individual from template; then attach the project with this individual only)?
